### DISCUSSION TITLE
Error when making requests to pb from browser

### DISCUSSION THREAD

**Discussion by sveltarne** [pocketbase/pocketbase#1110] [Q&A] [answered]
When i send a request from my webapp to pb, it gives this error:
![image](https://user-images.githubusercontent.com/114943151/203851495-c6250bf4-d1f6-48e4-b7b7-5a472b4f6797.png)

Any ideas what i might be doing wrong? You can check out the repo at https://github.com/sveltarne/observer/

**Comment by ganigeorgiev** [comment, marked as answer]
I'm not sure what could be the reason of the above error. By default PocketBase allows all origins unless you have specified a domain using the `--origins` flag.

If you have a reverse proxy (eg. nginx) then it is also possible that it is overwriting the default response headers.

What version of the PocketBase binary you are using (you can find it at the right-bottom corner in the Admin UI)? Note that PocketBase v0.8.0 is compatible only with JS SDK v0.8.0.

From the linked repo it seems that you are using JS SDK v0.7.4, which means that you must use a PocketBase v0.7.x executable.
